bot.py

```diff
@@ -20,10 +24,13 @@ async def on_ready():
 
 @client.command()
 async def actualizar(ctx):
-    await ctx.send("Datos actualizados :blush:")
-    # this code below allows the bot to restart itself after the command, might be useful in the future but in the meantime is not
+    notifications = json.loads(github_wrapper.getNotifications(USER_TOKEN))
```

Comment:
getNotifications will ask the API for a notifications **array**. Even though this array is JSON, it still is plain text. We need to use `json.loads` to parse that JSON text and convert it into array with nested dictionaries in python data structure, so we can work with this data as more complex variables in python. Finally, that's the content of `notifications`: an array of dictionaries, each dictionary represents a notification object.